plugins/vpp/ifplugin/vppcalls/nat_vppcalls.go

```diff
@@ -195,6 +201,13 @@ func (handler *NatVppHandler) handleNat44StaticMapping(ctx *StaticMappingContext
 		handler.stopwatch.TimeLog(nat.Nat44AddDelStaticMapping{}).LogTimeEntry(time.Since(t))
 	}(time.Now())
 
+	// Check tag length
+	tag := []byte(ctx.Tag)
+	if len(tag) > maxTagLen {
+		return errors.Errorf("static mapping label %s has '%d' bytes, max allowed is %d",
```

Comment:
Does not seem very useful to put quotes around a number, perhaps you intended to put it around the label string?